src/pages/black-friday.tsx

```diff
@@ -5,58 +5,98 @@ import HeroSection from '@/components/black-friday/HeroSection';
 
 import SuiteSection from '@/components/black-friday/SuiteSection';
 import CtaSection from '@/components/black-friday/CtaSection';
-import FeatureSection from '@/components/black-friday/FeatureSection';
 import PlatformSection from '@/components/black-friday/PlatformSection';
 import TestimonialsSection from '@/components/black-friday/TestimonialsSection';
 import FaqSection from '@/components/black-friday/FAQSection';
 import BestStorageSection from '@/components/black-friday/BestStorageSection';
-import LoginBFBanner from '@/components/banners/LoginBFBanner';
-import FooterSection from '@/components/black-friday/FooterSection';
+import { MinimalFooter } from '@/components/layout/footers/MinimalFooter';
+import { TextAndCardsGroupColumnSection } from '@/components/shared/components/TextAndCardsGroupColumnSection';
+import { Eye, Fingerprint, LockKey, ShieldCheck } from '@phosphor-icons/react';
+import TopBanner from '@/components/banners/TopBanner';
 
 const BLACK_FRIDAY_METATAG_ID = 'black-friday';
 
-const BlackFriday = ({ lang, metatagsDescriptions, langJson, navbarLang }) => {
+const BlackFriday = ({ lang, metatagsDescriptions, langJson, navbarLang, footerLang}) => {
   const metatags = metatagsDescriptions.filter((desc) => desc.id === BLACK_FRIDAY_METATAG_ID);
-
+  const groupCards = [
+    {
+      icon: ShieldCheck,
+      title: langJson.blackFriday.FeatureSection.feature1.title,
+      description: langJson.blackFriday.FeatureSection.feature1.subtitle1,
+    },
+    {
+      icon: LockKey,
+      title: langJson.blackFriday.FeatureSection.feature2.title,
+      description: langJson.blackFriday.FeatureSection.feature2.subtitle1,
+    },
+    {
+      icon: Eye,
+      title: langJson.blackFriday.FeatureSection.feature3.title,
+      description: langJson.blackFriday.FeatureSection.feature3.subtitle1,
+    },
+    {
+      icon: Fingerprint,
+      title: langJson.blackFriday.FeatureSection.feature4.title,
+      description: langJson.blackFriday.FeatureSection.feature4.subtitle1,
+    },
+  ];
   return (
     <Layout title={metatags[0].title} description={metatags[0].description} segmentName="Black Friday">
-      <Navbar
-        lang={lang}
-        darkMode={true}
-        isBlackfriday={true}
-        textContent={navbarLang}
-        cta={['Hide Login']}
-        isLinksHidden
-      />
-      <LoginBFBanner />
+      <TopBanner />
```

Comment:
Why? You can remove this one and use the banner we have in `Layout.tsx`. Then modifying the shouldShowBanner constant like:
`const shouldShowBanner = pathname === '/black-friday' && dialogIsOpen(GlobalDialog.TopBanner);`

You should be able to show the banner only for the black-friday LP.